src/components/Table/TableHeaderCell/TableHeaderCell.module.scss

```diff
@@ -0,0 +1,59 @@
+.tableHeaderCell {
+  background-color: var(--primary-background-color);
+  padding: 9px 8px;
+  padding-inline-start: 16px;
+  color: var(--secondary-text-color);
+  font: var(--font-text-small-normal);
```

Comment:
We don't need to use the font token for a container if it's already wrapped with the Text component